public/app/features/plugins/admin/types.ts

```diff
@@ -265,6 +266,7 @@ export enum PluginTabIds {
   USAGE = 'usage',
   IAM = 'iam',
   CHANGELOG = 'changelog',
+  RIGHTPANEL = 'right-panel',
```

Comment:
Why not just call this `PLUGINDETAILS`? I feel that `RIGHTPANEL` makes sense now but in a year when you don't have this PR fresh in mind the naming is a bit confusing.